monitoring/definitions/github_proxy.go

```diff
@@ -37,6 +37,20 @@ func GitHubProxy() *monitoring.Container {
 							PossibleSolutions: `Try restarting the pod to get a different public IP.`,
 						},
 					},
+					{
+						{
+							Name:            "github_proxy_waiting_requests",
+							Description:     "number of requests waiting on the global mutex",
+							Query:           `github_proxy_waiting_requests`,
+							DataMayNotExist: true,
+							Warning:         monitoring.Alert().GreaterOrEqual(100).For(5 * time.Minute),
+							PanelOptions:    monitoring.PanelOptions().LegendFormat("requests waiting"),
+							Owner:           monitoring.ObservableOwnerCloud,
+							PossibleSolutions: `
+                                - **Check github-proxy logs for network connection issues.
+                                - **Check github status.`,
```

Comment:
```suggestion
								- **Check github-proxy logs for network connection issues.
								- **Check github status.`,
```

(seems the generator expects tabs)